### PR TITLE
Upgrade to Elm 0.19.0

### PR DESCRIPTION
**Pull request by ryanolsonx**
See: #68

### CODE REVIEW

File: src/elm/Main.elm [hunk 1]
```diff
@@ -1,63 +1,75 @@
...
+                ]
+            ]
         ]
-      ]
-    ]
-  ]
-
-
--- CSS STYLES
-styles : { img : List ( String, String ) }
```
Comment (ryanolsonx) [inline src/elm/Main.elm:56]: This had to be removed because of how the api accepts `style` now. It's not the same in 0.19.0.

**Comment by ryanolsonx** [comment]
Don't merge this yet. I need to do some testing on the dependencies upgraded.

**Comment by ryanolsonx** [comment]
ping

**Comment by ryanolsonx** [comment]
@eeue56

**Comment by ryanolsonx** [comment]
So much for an Elm community repository. Screw this.

**Comment by atomkirk** [comment]
@ryanolsonx do you guy have a new repo where you keep it up to date?

**Pull request closed without merging**